### DISCUSSION TITLE
Cuckoo videos soon

### DISCUSSION THREAD

**Discussion by truecuckoo** [ctag-fh-kiel/ctag-tbd#37] [General]
Hello community of tbd. I'm a synth YouTuber going under the name True Cuckoo. I'll be using the TBD a bit this coming fall in my Eurorack setup. I'm new to GitHub so I don't really know how to use it.

I might have feedback during my use of it. But just hoping to loving it, really.

Just wanted to give you all heads up. In case there's a sudden spike of interest in TBD due to it being featured in some videos, it'd be great if everyone is ready to handle that.

Might be motivating too.

I'm gonna seriously try both the Rompler and the Wavetable stuff, which at first glance seems like a great place to start for me.

Hoping to get to know TBD a bit more in depth soon.

Cheers
Cuckoo

**Comment by ctag-fh-kiel** [comment]
Hi Cuckoo, 

Great to hear from you and here to help :) Awesome that TBD is of interest to you! 

Note that we have been working on a TBD mk2 for Eurorack with more (and better) CVs + TRIGs, including USB midi. Also, the community of [tangible waves](https://www.tangiblewaves.com/news/ae-tbd-is-a-new-digital-multifunction-module) will offer a mini version of TBD (mono, no WIFI) for their modular systems.

In case you only want to focus on Rompler and Wavetable, you can declutter your firmware by removing unused plugins and increase your sample rom significantly. You may want to check our [cloud-based custom firmware compiler](https://fxwiegand.github.io/tbd-cloud-compiler/). Note however, that a USB connection would be required to flash those custom firmwares.

Anyways let us know what you're up to :)

Best, Robert

**Reply by truecuckoo** [reply]
Thanks for the update. I'm sure the mk2 will be great. I had ￼a rather long session with mk1 when I first received it, a while ago. And I like the idea a lot. But I found it to be a little bit all over the place in terms of consistency, of course due to the open nature of it. So I think it's best for me to focus on a few favourite engines for my setup. Down the line though, I think with more inputs, to be able to hook up any parameter to be controlled independently, perhaps with midi too, will unleash the true potential. Perhaps even create the mk2 with an expander port for an optional CV input expander module?

**Reply by ctag-fh-kiel** [reply]
The mk2 will support:
- 2 pitch CV in -1.5..5.5V
- 4 bi-polar CV in -5..5V
- 4 uni-polar CV 0..5V including pots for offset / control without patch cords
- 4 pots for control (no CV jacks)
- 6 trig ins
- usb midi (device), where 2 channels are mapped note on/off, mod, bend to internal CVs / trigs
- Oled display with menu button to recall favorites --> in a live session there is no need to access the web ui, you can recall plugin + preset combinations stored in favorites

Currently all of that except 2 pots fit into 22HP, I think a slimmer version with custom break outs especially DIY friendly for building own hardware UIs / frontpanels would be a great target :) Or a bigger version with plenty of space and large knobs for better UX.

Will keep you posted and happy to receive feedback!

**Reply by ctag-fh-kiel** [reply]
> But I found it to be a little bit all over the place in terms of consistency, of course due to the open nature of it. So I think it's best for me to focus on a few favourite engines for my setup. 

Yes, fully agreed, TBD can be anything, it doesn't follow the concept "one module, one function" rather than "one module, any function" and the open nature certainly adds to that :) Focussing on favorite engines makes sense. With Rompler (sample playback) and WTOsc (wavetable oscillator with custom wavetables) there are two nice powerful basic engines for standard applications. I personally like to use the module a lot for Reverb (Progen Reverb) and FX (Antique). One can internally daisy chain CH0 to CH1 and use e.g. Rompler's output fed into the input of Antique.

**Comment by truecuckoo** [comment]
I've started messing with my system now. Posted a video just now but I didn't go in-depth. More to come.
I think a MK2 with more inputs is very welcomed, but at the cost of the width though..

I'm having my first proper try with Rompler and wave files. The file browser didn't work in Safari, so I had to use Chrome. When uploading sounds it's as if the Slices take up two slice slots. I only uploaded 40 short drum samples, but it looks like they're spread over 3-4 banks. So each slice is seemingly placed on two slices? So effectively a bank only contains about 16 unique sounds, instead of 32. Is this anything you guys are familiar with? And sorry for using this as a support channel.

**Reply by ctag-fh-kiel** [reply]
Hi there, great you've been giving Rompler a try. Your audio files are stereo, if you use stereo files, left and right channel are placed in subsequent slots. If you desire to use only a single slot, you have to use mono files for upload.

Hope this helps :)

**Reply by ctag-fh-kiel** [reply]
Ah, the reason for the stereo behaviour is that if you want to play stereo samples you can use two Rompler instances (CH0+CH1) with adjacent slices, playback will then be your original stereo sample.

**Reply by truecuckoo** [reply]
Ahhh! I get it! It'd be helpful if this was indicated in the browser already in a future update.

**Reply by ctag-fh-kiel** [reply]
Makes sense, will add it in the next release...

**Reply by truecuckoo** [reply]
Gotta say that pitch tracking is very wobbly for me. I did a calibration, but it's still wobbly. Trying out the WTO and I don't think it's really working well for melodic stuff for me. I think rhythmical and FX is probably more the territory for it?

**Reply by ctag-fh-kiel** [reply]
Yeah, pitch tracking could be better, that's a limitation of using the microcontrollers internal ADC unfortunately. 
It should work fine for like 3 octaves, full five octaves are a stretch, if you desire perfect pitch tracking.
Some external CV controllers offer slope compensation that helps (e.g. the analog4).
This shortcoming will be address in a mk2, however at a higher cost, as better components will need to be used.

**Reply by ctag-fh-kiel** [reply]
Ah, truecuckoo, I forgot to mention: 
PLEASE try the pitch quantizer in WTOSC, that should help you a lot! 
I added it especially to improve the pitch tracking and mitigate wobble. You can set it to '1' for semitone quantization.
More info is in the [docs](https://docs.google.com/document/d/1c8mjxWjdiJNP0xpkU2CxRUp9av6V4W39wARJf3_SMSo/edit#heading=h.yoq2ogbsg2wo)

**Reply by truecuckoo** [reply]
Ah yeah, I tried that. I kinda hits right in between a lot, and ends up even more wobbly. The WTO sounded cool though.

**Reply by ctag-fh-kiel** [reply]
Did you use the pitch slider to tune in WTOSC on a root note? Only then pitch quantization should be off to a good start (it uses the root note, which needs to be tuned in the plugin).

**Reply by ctag-fh-kiel** [reply]
BTW Dual WTO on two channels with a phase offset + slightly different wavetables gives a great stereo feel to the sound;)
